### PR TITLE
GSIP-225 update mailing list references

### PR DESCRIPTION
**Pull request by petersmythe**
Linked to https://github.com/geoserver/geoserver/pull/7796

* [x] blog post about signing up to discourse
* [x] update community support page as best we can
* [x] what else is needed?

### CODE REVIEW

File: comm/index.html [hunk 1]
```diff
@@ -9,64 +9,84 @@ <h1>Community support</h1>
...
         <h3>Responsible Disclosure of Security Issues</h3>
     
         {% include disclosure.html %}
     </section>
 
+
+<a id="discourse"></a>
+    <h2>Discourse</h2>
+    <p>
+    The GeoServer-Users mailing list has migrated to <a href="https://discourse.osgeo.org/c/geoserver/users">Discourse</a>, an opensource web-based forum platform provided by OSGeo. All 20+ years of email archives are available to be searched and built upon, alongside other OSGeo greats such as QGIS, PostGIS, pgRouting, etc.</br>
```
Comment (jodygarnett) [inline comm/index.html:23]:
```suggestion
    The GeoServer User forum is hosted  on <a href="https://discourse.osgeo.org/c/geoserver/user">discourse</a> provided by the Open Source Geospatial Foundation. The forum contains all 20+ years of content from the archived Source Forge email list.</br>
```

File: comm/index.html [hunk 2]
```diff
@@ -9,64 +9,84 @@ <h1>Community support</h1>
...
+
     <section>
-        <h3>User List</h3>
+        <h3>User Group</h3>
```
Comment (jodygarnett) [inline comm/index.html:26]:
```suggestion
        <h3>GeoServer User Forum</h3>
```

File: comm/index.html [hunk 3]
```diff
@@ -9,64 +9,84 @@ <h1>Community support</h1>
...
+		</p>
+		<p>
             <strong>Do <i>not</i> report security vulnerabilities here.  See the <i>Responsible disclosure</i> instructions above.</strong>
+		</p>
+		<p>
+			If you are not in a position to communicate in public, please contact one of our <a href="../support">commercial support</a> vendors.
+
         </p>
+
+        <h4>Signing up</h4>
```
Comment (jodygarnett) [inline comm/index.html:41]:
```suggestion
        <h4>Discourse Sign Up</h4>
        
        <p>
        Discourse allows you to sign up using an account you already own:
        </p>
```

File: comm/index.html [hunk 4]
```diff
@@ -9,64 +9,84 @@ <h1>Community support</h1>
...
+		<p>	
+		Start here: <a href="https://discourse.osgeo.org">https://discourse.osgeo.org</a> > Sign Up.
+		</p>
+		
+        <h4>Transitioning from mailing lists?</h4>
+		If you want to keep on using email as the primary way of interacting with the GeoServer User list, you're going to have to follow these instructions, because the old SourceForge/mailman list will be switched off in a month's time/end Aug 2024.</br>
+		
         <ul>
-            <li>Sign up to <a href="https://lists.sourceforge.net/mailman/listinfo/geoserver-users">geoserver-users</a> email list on Source Forge. After subscribing you can post at geoserver-users@lists.sourceforge.net
+            <li>Log in, and navigate to GeoServer / Users.</li>
```
Comment (jodygarnett) [inline comm/index.html:60]:
```suggestion
            <li>Log in, and navigate to <a href="https://discourse.osgeo.org/c/geoserver/users">GeoServer / Users</a>.</li>
```

File: comm/userlist-guidelines.html [hunk 5]
```diff
@@ -1,44 +1,46 @@
 ---
 layout: default
-title: User list posting guidelines
+title: User group posting guidelines
 id: comm-ul-guidelines
 ---
 
-<h1>User list posting guidelines</h1>
+<h1>User group posting guidelines</h1>
```
Comment (jodygarnett) [inline comm/userlist-guidelines.html:7]:
```suggestion
<h1>User forum posting guidelines</h1>
```

File: comm/userlist-guidelines.html [hunk 6]
```diff
@@ -1,44 +1,46 @@
...
 <p class="lead">
-Regardless of whether you are just starting out with GeoServer or have been using it for years, we encourage you to subscribe to the <a href="http://lists.sourceforge.net/mailman/listinfo/geoserver-users" rel="nofollow">user list</a> and take an <b>active</b> part in the community. To help make the user list as useful and friendly as possible we ask you to observe the following code of conduct.
+Regardless of whether you are just starting out with GeoServer or have been using it for years, we encourage you to subscribe to the <a href="https://discourse.osgeo.org/c/geoserver/users" rel="nofollow">Discourse user group</a> and take an <b>active</b> part in the community. To help make the user group as useful and friendly as possible, we ask you to observe the following code of conduct.
```
Comment (jodygarnett) [inline comm/userlist-guidelines.html:9]:
```suggestion
Regardless of whether you are just starting out with GeoServer or have been using it for years, we encourage you to join to the <a href="https://discourse.osgeo.org/c/geoserver/users" rel="nofollow">GeoServer User forum</a> and take an <b>active</b> part in the community. To help make the forum as useful and friendly as possible, we ask you to observe the following code of conduct.
```

**Review by jodygarnett — CHANGES_REQUESTED** [review]
Thanks so much for starting this @petersmythe 

I have provided some suggestions that amount to:

Wording so this about use of the forum (not about migrating from the email list - new people won't know about the mailing list). 

Trying "GeoServer User Forum"  using forum rather than group. The end result is a forum right, it just happens that discourse forum software uses categories and groups to organize how topics are discussed....

File: comm/index.html (hunk 1 above)
Comment (petersmythe) [reply comm/index.html:23]: User or Users?

Discourse is capitalised: https://en.wikipedia.org/wiki/Discourse_(software)

SourceForge is one word

File: comm/index.html [hunk 7]
```diff
@@ -9,64 +9,84 @@ <h1>Community support</h1>
...
+
     <section>
-        <h3>User List</h3>
+        <h3>User Group</h3>
         <p>
-            This list is for <b>end users</b> of the GeoServer software and focused on topics 
-            related to normal GeoServer usage. While it's focused mainly about GeoServer itself it 
-            is also encouraged for users to post about topics around GeoServer, such as 
+            This group is for <b>end users</b> of the GeoServer software and focused on topics 
+            related to normal GeoServer usage. While it's focus is mainly about GeoServer itself, users
```
Comment (petersmythe) [inline comm/index.html:29]: its focus

**Comment by jodygarnett** [comment]
With discourse live this PR can be reviewed now

File: comm/index.html (hunk 1 above)
Comment (jodygarnett) [reply comm/index.html:23]: I cannot tell if it is GeoServer Users or Users as it is in consistent.

* User interface says "GeoServer" "User"
* URL says geoserver-users
* Email address says geoserver-user@discourse.osgeo.org

I asked for help here: https://discourse.osgeo.org/t/testing-new-topic-by-mail/49596/7

But I think we best use "geoserver-user" to match the email address.

[the remark this answers is not in the document]
Comment (petersmythe) [reply comm/index.html]: Removed this line

**Review by jodygarnett — APPROVED** [review]

**Pull request merged**